documentation/src/main/resources/pages/ditto/release_notes_340.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Release notes 3.4.0
+tags: [release_notes]
+published: true
+keywords: release notes, announcements, changelog
+summary: "Version 3.4.0 of Eclipse Ditto, released on 13.09.2023"
+permalink: release_notes_340.html
+---
+
+The fourth minor release of Ditto 3.x, Eclipse Ditto version 3.4.0 is here.
+
+This release is completely [IP (intellectual property) checked by the Eclipse Foundation](https://www.eclipse.org/projects/handbook/#ip)
+meaning that project code as well as all used dependencies were "[...] reviewed to ensure that the copyrights
+expressed are correct, licensing is valid and compatible, and that other issues have been uncovered and properly
+investigated."
+
+
+## Changelog
+
+Compared to the latest release [3.3.6](release_notes_336.html), no features were added. 
```

Comment:
This is not true .. E.g. the `documentDB` support is in 3.4.0, as well as other topics: https://github.com/eclipse-ditto/ditto/milestone/61

Reply:
Oh. My bad. I thought that it went in with the previous release.